web-app/src/app/app/jobs/[agentId]/components/jobs-table.tsx

```diff
@@ -14,10 +15,15 @@ interface JobsTableProps {
 export default function JobsTable({ jobs }: JobsTableProps) {
   const t = useTranslations("App.Jobs.JobsTable");
   const dateFormatter = useFormatter();
+  const router = useRouter();
 
   return (
     <DataTable
       columns={getColumns(t, dateFormatter)}
+      rowOnClick={(row) => () => {
+        router.push(`/app/jobs/${row.agentId}/${row.id}`);
```

Comment:
Please let's use the Route Types.

Reply:
Not sure what you mean, can you give an example?

Reply:
if you mean the enum, it is a dynamic type

Reply:
Possible fix https://github.com/masumi-network/sokosumi/pull/248/commits/53024cd0834714636ddeac933615a3b0ec17fd9a